docs/usage/basic_usage.md

```diff
@@ -0,0 +1,256 @@
+sdk.get_jobs(filters=JobFilters(batch_id="batch_id"))
+
+# Get the first 100 jobs in error from a specific project
+sdk.get_jobs(filters=JobFilters(status=JobStatus.ERROR, project_id="project_id"))
+
+# Get two jobs using two ids
+sdk.get_jobs(filters=JobFilters(id=["job_id_1", "job_id_2"]))
+
+# Get the first 20 cancelled jobs created in a given period from a specific list of users
+sdk.get_jobs(limit=20,
+             filters=JobFilters(status=JobStatus.CANCELED, start_date=datetime(...), end_date=datetime(...),
+                                user_id=["user_id_1", "user_id_2"]))
+
+# Get the total number of jobs matching the filters
+sdk.get_jobs(pagination_params=PaginationParams(offset=0)).total
+
+# Get the first 300 jobs, no filters applied
+jobs = []
+jobs.extend(sdk.get_jobs(pagination_params=PaginationParams(offset=0)).results)
+jobs.extend(sdk.get_jobs(pagination_params=PaginationParams(offset=100)).results)
+jobs.extend(sdk.get_jobs(pagination_params=PaginationParams(offset=200)).results)
+
+```
+
+## Retry a batch of jobs
+
+It is possible to retry a selection of jobs from a CLOSED batch with the `rebatch` method.
+
+```python
+from datetime import datetime
+from pasqal_cloud import RebatchFilters, JobStatus
+
+# Retry all jobs from a given batch
+sdk.rebatch(batch_id)
+
+# Retry the first job of a batch
+sdk.rebatch(batch_id, RebatchFilters(id=batch.ordered_jobs[0].id))
+
+# Retry all jobs in error
+sdk.rebatch(batch_id, RebatchFilters(status=JobStatus.ERROR))
+
+# Retry cancelled jobs created in a given period
+sdk.rebatch(batch_id, RebatchFilters(status=JobStatus.CANCELED, start_date=datetime(...), end_date=datetime(...)))
+
+# Retry jobs that have a run number between 5 and 10
+sdk.rebatch(batch_id, RebatchFilters(min_runs=5, max_runs=10))
+```
+
+## Retry a job in an open batch
+
+It is possible to retry a single job in a same open batch as an original job using `batch.retry`.
+The batch must be open in order for this method to work.
+
+```python
+
+batch = sdk.create_batch(..., open=True)
+
+batch.retry(batch.ordered_jobs[0])
+
+# Like for adding a job you can choose to wait for results.
+batch.retry(batch.ordered_jobs[0], wait=True)
+```
+
+## Cancel a list of batches
+
+It is possible to cancel a list of batches with the `cancel_batches` method by
+providing a list of batch ids.
+
+```python
+sdk.cancel_batches(batch_ids=[...])
+```
+
+## Cancel a list of jobs
+
+It is possible to cancel a selection of jobs with the `cancel_jobs` method, using the CancelJobFilters class.
+
+Here are few examples of how to use it:
+
+```python
+from datetime import datetime
+from pasqal_cloud import CancelJobFilters
+
+# Cancel two specific jobs
+sdk.cancel_jobs(batch_id="batch_id", filters=CancelJobFilters(id=["job_id_1", "job_id_2"]))
+
+# Cancel jobs created in a given period of time
+sdk.cancel_jobs(batch_id="batch_id", filters=CancelJobFilters(start_date=datetime(...), end_date=datetime(...)))
+```
+
+## Create a workload
```

Comment:
imo we should have a page for batches/jobs and another for workloads

Reply:
I've done it, and tried to add some structure to the workload page. I still think how we describe Workloads is confusing at best ("A workload is a unit of work to be executed on Pasqal Cloud Services infrastructure" :shrug:), and that we should dive more into the fundamental difference between batches/jobs and workloads.

But let's do that later.